react-frontend/src/core-components/video/VideoComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const VideoComponent = ({ className, src, controls, height, width, title }) => (
```

Comment:
Where do we need a video component? if not remove it
cc: @sahilbhatia

Reply:
In log in presentation component have slide so we we will use image and video inside slide.